www/nginx/src/opnsense/mvc/app/models/OPNsense/Nginx/Nginx.xml

```diff
@@ -1,6 +1,6 @@
 <model>
   <mount>//OPNsense/Nginx</mount>
-  <version>1.20.1</version>
+  <version>1.21.0</version>
```

Comment:
should be the latest nginx version that needed a migration. so this should be 1.24.0 in your case. So the migration is 1_24 as well.

Reply:
CARP is a bad example, you need something like PPPoE which may disconnect every 24h or something where you may get another IP assigned.